event/src/routes/event/get_event_sessions.rs

```diff
@@ -0,0 +1,50 @@
+use poem::web;
+use poem_openapi::{param::Path, payload};
+
+use crate::{database::Database, entities, error::ErrorResponse};
+
+#[derive(poem_openapi::ApiResponse)]
+pub enum Response {
+    #[oai(status = 200)]
+    Ok(payload::Json<entities::Session>),
+}
+
+#[derive(poem_openapi::ApiResponse)]
+pub enum Error {
+    #[oai(status = 400)]
+    BadRequest(payload::Json<ErrorResponse>),
+
+    #[oai(status = 404)]
+    NotFound(payload::Json<ErrorResponse>),
+
+    #[oai(status = 500)]
+    InternalServer(payload::Json<ErrorResponse>),
+}
+
+impl crate::routes::Routes {
+    pub async fn _get_event_sessions(
```

Comment:
I believe this should actually be `list_event_sessions` and the query actually should be `fetch_many`

Reply:
it actually should be `fetch_all` because `fetch_many` returns an async stream